dev-packages/node-integration-tests/suites/tracing/google-genai/scenario.mjs

```diff
@@ -0,0 +1,138 @@
+import { instrumentGoogleGenAIClient } from '@sentry/core';
+import * as Sentry from '@sentry/node';
+
+class MockGoogleGenAI {
```

Comment:
l: This can be done in a follow up, but it would be great to also have one test that tests the auto instrumentation, so without manually wrapping the client :)